boot/freeldr/freeldr/arch/i386/irqsup.S

```diff
@@ -0,0 +1,33 @@
+/*
+ * PROJECT:     FreeLoader
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Interrupt handling
```

Comment:
but interrupt handling for _**InPort mouse recognition**_? (If so, maybe "moudtect.S" or "inport.S" or sthg like that)

Reply:
I'm going to use this code for CardBus detection as well.